span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/SqlFieldsGenerator.java

```diff
@@ -71,7 +71,16 @@ protected void maybePopulateSqlUrlForOtelSpan(
       final Map<String, AttributeValue> attributeFieldMap) {
     if (DbSemanticConventionUtils.isSqlTypeBackendForOtelFormat(attributeFieldMap)) {
       Optional<String> sqlUrl = DbSemanticConventionUtils.getBackendURIForOtelFormat(attributeFieldMap);
-      sqlUrl.ifPresent(s -> eventBuilder.getSqlBuilder().setUrl(s));
+      if (sqlUrl.isPresent()) {
```

Comment:
Can't we use `getSqlURI` form DbSemanticsConvention?

Reply:
That accepts Event object, we have an attribute map here. I can create one more method there. Should I?

Reply:
Yes, it's a common code, and it's difficult to maintain.